README.md

```diff
@@ -15,16 +15,12 @@ discussion on scoverage.
 
 Make sure your SBT version in project/build.properties:
 ```
-sbt.version = 0.13.17
-```
-or
-```
-sbt.version = 1.1.1
+sbt.version = 1.5.1
```

Comment:
Can you expand a bit on why 1.5.1 is now the default minimum version?

Reply:
While this may not be accurate its all I tested with.

Reply:
I am just giving 1.1.1 a try.

Reply:
1.1.1 gives me some error, but 1.2.8 works. Will set this as the minimal Version for now.

Reply:
Perfect.